src/content/docs/codestream/start-here/codestream-new-relic.mdx

```diff
@@ -0,0 +1,228 @@
+* [New Relic user key](https://one.newrelic.com/launcher/api-keys-ui.api-keys-launcher)
+* [CodeStream and New Relic connection](#connect)
+* [Data being reported to New Relic via APM monitoring](https://one.newrelic.com/launcher/nr1-core.explorer?pane=eyJuZXJkbGV0SWQiOiJucjEtY29yZS5saXN0aW5nIn0=&cards[0]=eyJuZXJkbGV0SWQiOiJucjEtaW5zdGFsbC1uZXdyZWxpYy5ucjEtaW5zdGFsbC1uZXdyZWxpYyIsImFjdGl2ZUNvbXBvbmVudCI6IlZUU09FbnZpcm9ubWVudCIsInBhdGgiOiJndWlkZWQifQ==)
+* [A workload for errors inbox](/docs/new-relic-one/use-new-relic-one/workloads/use-workloads/#create)
+* [An error](/docs/apm/apm-ui-pages/errors-inbox/errors-inbox/#error-details)
+* [For APM errors, your repository's commit hash and release tag](#apm)
+
+New Relic errors inbox is a single place to proactively detect, prioritize, and take action on your errors before they impact customers. With CodeStream, you can jump from an error directly to the offending code in your IDE.
+
+<Callout variant="tip">
+    Limited to APM errors.
+</Callout>
+
+From **[one.newrelic.com/](https://one.newrelic.com/)**, go to **Errors Inbox**, click a stack trace error, then click **Open in IDE**.
+
+## APM errors and CodeStream [#apm]
+
+To use CodeStream's **Open in IDE** with your APM stack trace errors, configure your APM agent with your application's [commit sha](https://git-scm.com/book/en/v2/Git-Tools-Revision-Selection)
+and/or your [release tag](https://git-scm.com/book/en/v2/Git-Basics-Tagging) associated with the running version of your software.  
+
+CodeStream only needs the first seven characters of your commit sha (for example, `734713b`) to make this connection, but you can include the entire sha.  
+
+Alternately, you can use a release tag (such as `v0.1.209` or `release-209`) for CodeStream to find the correct version of your code.
+
+For New Relic APM, the commit and/or release tag (`tags.commit` and `tags.releaseTag`) are added as attributes on `Transaction` and `TransactionError` events. The APM agent protocol provides an easy way to set this up via environment variables.  
+
+We recommend setting one or both of these variables as part of your build pipeline.
+* `NEW_RELIC_METADATA_COMMIT` - The commit sha. You can include the whole thing or only the first seven characters.
+* `NEW_RELIC_METADATA_RELEASE_TAG` - A release tag (such as `v0.1.209` or `release-209`). This has the advantage of being human readable.
+
+## Give us your git repository's URL [#repo-url]
+
+In order to link your data in New Relic to your code, we need to know the URL for your git repository.  
+
+The URL can be in either the SSH or HTTPS format:
+* `git@github.com:newrelic/beta-docs-site.git`
+* `https://github.com/newrelic/beta-docs-site.git`
+
+<Callout variant="caution">
+It's possible to add the same GitHub repository more than once, if you're different protocols. The UI warns you about this, but won't prevent you from doing so.
```

Comment:
I wasn't entirely sure what the missing word was here, so I didn't just add it: 
"if you're different protocols" - should be something like "if you're using different protocols" but maybe setting or creating?

Reply:
It was "using". I wrote it in my mind :)